int32 id
float32 distance
geometry_msgs/Vector3 direction
 
